packages/client/e2e/_utils/page.ts

```diff
@@ -216,10 +216,6 @@ export async function bypassVerifyEmailScreen({
 
     await page.request.get(`${SERVER_ENDPOINT}/test/bypass-email-confirmation`);
 
-    //FIXME TMP should be removed after email polling
-    await page.reload();
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏It seems to be a little deletion, but it is due to a massive improvement.